scio-avro/src/main/scala/com/spotify/scio/avro/AvroIO.scala

```diff
@@ -141,21 +141,32 @@ sealed trait AvroIO[T] extends ScioIO[T] {
       .withMetadata(metadata.asJava)
 }
 
-final case class SpecificRecordIO[T <: SpecificRecord: ClassTag: Coder](path: String)
+final case class SpecificRecordIO[T <: SpecificRecord: ClassTag: Coder](paths: Iterable[String])
```

Comment:
Could be defined this way to avoid breaking back-compat:

```suggestion
final case class SpecificRecordIO[T <: SpecificRecord: ClassTag: Coder](paths: String*)
```

Alternatively:

```suggestion
final case class SpecificRecordIO[T <: SpecificRecord: ClassTag: Coder](path: String, paths: String*)
```

which guaranties at least 1 path is passed

Reply:
Since we have `apply` function which takes only one string argument. this won't introduce any back-compatibility issue.  But of course, this is another way to do it. the only downside with varargs approach is, it takes away passing any additional argument at the end of the constructor if we might need it in the future.

Reply:
Yeah sorry I did not see the added `apply`. You can disregard this comment ^^